### ISSUE TITLE
URGENT problem

### ISSUE THREAD

**Issue by isomemo** [Pandora-IsoMemo/ReSources#130]
There is an urgent basic problem with something that was previously working (simplest possible model). A colleague saved the ReSources model below but when trying to load it, we both get the error "Empty tables"

Furthermore, the model cannot be run. The error "Not valid source values" is issued, although the source values are all numeric.

Please give top priority to fix this! I also would like to discuss what else may be broken.

The file is zipped since GitHub does not accept the .resources extension.

[2024-09-16_18-15-33_model.zip](https://github.com/user-attachments/files/17016313/2024-09-16_18-15-33_model.zip)

**Comment by arunge** [comment]
I do receive following when uploading the model:

![image](https://github.com/user-attachments/assets/fda4771c-ab55-438c-b515-2108c348fc56)

and when debugging:

```R
DEBUG [2024-09-17 11:22:33] Entering observe() (set values$xxxNames)
Warnung: Error with source names
 error: Objekt 'newValue' nicht gefunden
Error : 
```

I am checking what is missing.

**Comment by arunge** [comment]
@isomemo The error 'newValue not found' occurs only on beta, I will fix this with the next version.

However, I checked the file you attached above. The input tables do exist in the object that stores inputs, data and the model, but they only contain one line with the value `NA`. That is why, we get the message for 'Empty tables'. And that is why, no input data is loaded. Because in the object there is none:

![image](https://github.com/user-attachments/assets/aae73e7e-cb2f-4bf0-927a-471effa53a80)

I also cannot see any source values and wonder why you said you have numeric values:

![image](https://github.com/user-attachments/assets/494e1d62-b675-4b84-bd93-e9587d7d1b7a)

The error 'No valid source values' occurs because no source data is available since the content is `NA`.

![image](https://github.com/user-attachments/assets/932abc0e-c718-433e-afc3-0be63c4c70cc)

Did you send me exactly the file that you tried to upload? The name of the file (`2024-09-16_18-15-33_model.resources`) inside the zip from above indicates that you downloaded a new model object after trying to load the other one. But this file does not contain the input data. Is it possible to provide me the original file that you tried to upload?

**Comment by isomemo** [comment]
@arunge the file was saved after all values were inputed. It was done by a colleague during a live Zoom call and it was clear that everything was done correctly. Can you please check if now saving/loading works fine?

**Comment by arunge** [comment]
@isomemo Yes, I just checked the [main version](https://isomemoapp.com/app/resources) (still 24.5.0.1)
- after running a model it is possible to download a model
- after restarting the app (to ensure that previous data was removed) I can import the model without any issues. Data, inputs and model output is loaded. 

However, the fix for the issue on beta must still be deployed. I will close here as soon as this is done.

**Comment by arunge** [comment]
Regarding from above
> However, the fix for the issue on beta must still be deployed. I will close here as soon as this is done.

The Fix is online with version 24.09.0 on [ReSources beta](https://isomemoapp.com/app/resources-beta). I am closing here.
